### DISCUSSION TITLE
reCaptcha in MAUI

### DISCUSSION THREAD

**Discussion by YordanUzunov** [CommunityToolkit/Maui#988] [General]
Hey guys, I'm struggling with implementing reCaptcha in MAUI... Is there any way?

**Comment by pictos** [comment]
You can integrate Blazor components in MAUI, so if there's a reCaptcha component that runs on Blazor you should be able to integrate that on MAUI

**Comment by VladislavAntonyuk** [comment]
I will create a sample later this week.

**Reply by YordanUzunov** [reply]
Thank you for the initiative!

**Comment by vhugogarcia** [comment]
Hello @YordanUzunov and friends,

I just published an article on my tech blog about how to implement reCaptcha in .NET MAUI. I hope it helps you! This implementation is the one I'm using in a personal project and it works very good.

[https://dev.to/vhugogarcia/google-recaptcha-in-net-maui-56m0](https://dev.to/vhugogarcia/google-recaptcha-in-net-maui-56m0)

Regards,
Victor

**Comment by VladislavAntonyuk** [comment]
An alternative implementation for .NET MAUI and .NET MAUI Blazor: https://vladislavantonyuk.azurewebsites.net/articles/Integrating-captcha-into-a-.NET-MAUI-application. 
It looks a bit simpler and doesn't require DependencyService.

**Comment by CliffAgius** [comment]
Closed as per the Community Stand-up discussion here: https://youtu.be/N9wMcBP4jtg?t=2889
